recipe/conda_build_config.yaml

```diff
@@ -4,3 +4,12 @@ cxx_compiler_version:      # [linux]
   - 12                     # [linux]
 arm_variant_type:  # [aarch64]
   - sbsa           # [aarch64]
+
+cross_target_platform:
+  - linux-64
+  - linux-aarch64
+  - linux-ppc64le
+cross_target_impl_platform:
+  - linux-64
+  - linux-aarch64
+  - linux-ppc64le
```

Comment:
This becomes a cartesian product of `cross_target_platform` & `cross_target_impl_platform`. Ideally all values from `cross_target_impl_platform` are added to the variant files (as opposed to being split out one per variant file)

Another way we might be do this is to convert this into a nested list, which iterate over in the recipe

Reply:
No longer using `cross_target_impl_platform`